src/pybind/mgr/dashboard/tests/test_access_control.py

```diff
@@ -7,6 +7,7 @@
 import time
 import unittest
 from datetime import datetime, timedelta
+from os import linesep
```

Comment:
this is always going to be `\n`. AFICT `os.linesep` is meant to ease cross-platform development by being different on windows systems. but that is never going to be the case.

I'm :+1: for keeeping `'\n'` hard coded

Reply:
> this is always going to be `\n`. AFICT `os.linesep` is meant to ease cross-platform development by being different on windows systems. but that is never going to be the case.
> 
> I'm  for keeeping `'\n'` hard coded

@sebastian-philipp If this is the general consensus, OK. But do you mind to put this in the Ceph Style Guide?
As currently nothing is preventing any contributor from using platform-agnostic code like ```linesep``` here:
https://github.com/ceph/ceph/blob/master/CodingStyle#L141

Reply:
My problem with this code is, it is misleading. `linesep` changes based on the platform the MGR is running on and this hardly changes. This has no effect on which OS was used to  generate the inbuf. If you want to have this platform-agnostic, you'll have to handle both `\n` and `\r\n` on all platforms. Imagine you have a MGR running on a windows machine and someone is using a linux client.

Reply:
I agree with you that we should add ```\r\n``` sanitization (who knows where the file was generated), so then it makes sense to make both sanitizations explicit.